kyaml/filesys/errors.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package filesys
+
+import "sigs.k8s.io/kustomize/kyaml/errors"
+
+var ErrNotDir = errors.Errorf("invalid directory")
```

Comment:
`ErrNotDir` to me implies that it's not a directory, which sounds different to me than an invalid directory. I'm not sure which you intend it to be here, but let's make the name consistent with the message. Either 

```
var ErrNotDir = errors.Errorf("not a directory")
```

or 

```
var ErrInvalidDir = errors.Errorf("invalid directory")
```

Reply:
Ah, I see! Thanks for explaining. Could we then change the message to say `var ErrNotDir = errors.Errorf("not a valid directory")`?